index.src.html

```diff
@@ -1380,6 +1380,28 @@ <h4 id="body-extraction" algorithm>
           6.  Set |Content-Type| to
               `application/x-www-form-urlencoded;charset=UTF-8`.
 
+          Otherwise, |data| is a generic object, so:
+
+          1.  If |data| has a property named like the |credential|'s
+              {{PasswordCredential/idName}} or
+              {{PasswordCredential/passwordName}}, <a>throw</a> a `TypeError`
+              exception.
```

Comment:
All copies will always trigger getters. Not sure what you mean about modification though. I don't have enough context to understand why the code snippet above might cause an attack; is the spec meaning to actually modify user-provided objects? That does seem not great.

Maybe structured clone, or JSON.stringify + JSON.parse, would be helpful here.